components/virtual-node/virtual-node-write-service-impl/build.gradle

```
@@ -7,6 +7,7 @@ description "Configuration Write Service Impl"
 
 dependencies {
     api project(':libs:virtual-node:virtual-node-info')
+    implementation project(path: ':libs:external-messaging')
```

you don't need the path keyword here. 
```suggestion
    implementation project(':libs:external-messaging')
```

---

Yes, you are right. I though I had removed all of them.